EventSourcing.Cosmos/StoredProcedures/StoredProcedures.cs

```diff
@@ -0,0 +1,123 @@
+namespace EventSourcing.Cosmos;
+
+public static class StoredProcedures
+{
+    // String of the JS function deleteAggregateAll that is uploaded as a stored procedure
+    public static string DeleteAggregateAll = @"/**
```

Comment:
Maybe it is actually better to use the .js file directly for uploading the stored procedure? That way no errors will be made in the future forgetting to update the .js string? What do you think?

Reply:
Ive also added a test to Assert.Equal the static string with the text in the js file